build/ck.build.s390x

```diff
@@ -0,0 +1,2 @@
+CFLAGS+=-m64 -O0 -g
```

Comment:
These don't match production build flags elsewhere. How come -O0 is being used? If you're running into regression failures with optimization on, it probably indicates something is wrong in the `ck_pr` port below.